documentatie/flowcharts/date-entry-navigation.md

```diff
@@ -0,0 +1,149 @@
+# Data Entry Navigation
+
+## Questions
+- Where does "cache input" happen in these flows?
+- Is there a different warning icon for accepted versus not-accepted warnings? Or is it not possible to leave a page without accepting the warning? Also, accepting warnings is not yet included in any of the flows.
+
+
+## Render page
+
+Render happens based on last received API response.
+
+```mermaid
+flowchart TD
+
+    %% elements
+    flow-start([start])
+    flow-end([end])
+    illegal-state{{illegal state}}
+
+    show-error
+    error-icon-next-page
+    %% error-icon-prev-page is illegal state
+    error-icon-cur-page
+
+    show-warning
+    warning-icon-next-page
+    warning-icon-prev-page
+    warning-icon-cur-page
+
+    error-next-page{error-next-page?}
+    error-prev-page{error-prev-page?}
+    error-cur-page{error-cur-page?}
+
+    warning-next-page{warning-next-page?}
+    warning-prev-page{warning-prev-page?}
+    warning-cur-page{warning-cur-page?}
+
+    cur-page-submitted{cur-page-submitted?}
+
+    fill-cached-input
+
+    %% flow
+    flow-start --> error-prev-page
+
+    error-prev-page -- yes --> illegal-state
+    error-prev-page -- no --> error-next-page
```

Comment:
any next page could have an error. Only one, and there could not be more then one. But it does not have to be the immediate next page. Is this what you mean here?

Also: why do we need to know whether the next page has an error when rendering the current page? We need to know this to set the correct state on the items in the navigation menu, but then we need to know it for all pages, not just prev, current and next?

Reply:
prev-page is indeed any previous page. next-page is any next page. Will add the "any".